executor/prepared.go

```diff
@@ -397,6 +397,17 @@ func ResetStmtCtx(ctx context.Context, s ast.StmtNode) {
 		sessVars.PrevLastInsertID = sessVars.LastInsertID
 		sessVars.LastInsertID = 0
 	}
+	if sessVars.StmtCtx != nil {
```

Comment:
use a separate func to make the code clean.

Reply:
Could you remove these code to built func eval?

Reply:
@winkyao row_count relies on affectRows var and the last execute sql type.
I can not find this way to get rid of variable PrevAffectedRows.